x-pack/test/reporting/README.md

```diff
@@ -2,152 +2,29 @@
 
 ### Overview
 
-Reporting tests have their own top level test folder because:
-  - Current API tests run with `optimize.enabled=false` flag for performance reasons, but reporting actually requires UI assets.
-  - Reporting tests take a lot longer than other test types. This separation allows developers to run them in isolation, or to run other functional or API tests without them.
+Reporting tests have their own top level test folder because: the current API tests run with `optimize.enabled=false` flag for performance reasons, but reporting actually requires UI assets.
 
   ### Running the tests
 
-  There is more information on running x-pack tests here: https://github.com/elastic/kibana/blob/master/x-pack/README.md#running-functional-tests. Similar to running the API tests, you need to specify a reporting configuration file. Reporting currently has two configuration files you can point to:
+  You need to specify a reporting configuration file. Reporting currently has two configuration files you can point to:
   - test/reporting/configs/chromium_api.js
-  - test/reporting/configs/chromium_functional.js
+  - test/reporting/configs/generate_api.js
 
-  The `api` versions hit the reporting api and ensure report generation completes successfully, but does not verify the output of the reports. This is done in the `functional` test versions, which does a snapshot comparison of the generated URL against a baseline to determine success.
+  The tests hit the reporting API and ensure report generation completes successfully, but does not verify the output of the reports.
 
   To run the tests in a single command. :
-1. cd into x-pack directory.
-2. run:
+1. run:
   ```
-node scripts/functional_tests --config test/reporting/configs/[config_file_name_here].js
+node scripts/functional_tests --config x-pack/test/reporting/configs/[config_file_name_here].js
   ```
 
  You can also run the test server seperately from the runner. This is beneficial when debugging as Kibana and Elasticsearch will remain up and running throughout multiple test runs. To do this:
 
-1. cd into x-pack directory.
```

Comment:
I don't like having to do `cd` in my workspace, so I changed the instructions to add `x-pack` as a prefix to the config arguments